utils/deployment/exoscale/README.md

```diff
@@ -0,0 +1,88 @@
+# Exoscale Rain Deployment Tool
+
+This tool enables to simply create, deploy, manage and destroy computational resources at [Exoscale](https://www.exoscale.com/) cloud infrastructure.
+
+## Usage
+
+The tool is based on [Python Cloustack API client](https://github.com/exoscale/cs) and requires the following configuration in either `$HOME/.cloudstack.ini` or `./cloudstack.ini`:
+
+```
+[cloudstack]
+endpoint = https://api.exoscale.ch/compute
+key = cloudstack api key
+secret = cloudstack api secret
+# Optional ca authority certificate
+verify = /path/to/certs/exoscale_ca.crt
+# Optional client PEM certificate
+cert = /path/to/client_exoscale.pem
+```
+
+### Create a set of virtual machines (environment)
+
+`python3 exoscale.py create --help`
+
+```
+usage: exoscale.py create [-h] [--n N] [--name NAME] [--env ENV] --keypair
+                          {key1,key2}
+                          [--offering {micro,tiny,small,medium,large,extra-large,huge,mega,titan,gpu-small,gpu-huge}]
+                          [--zone {ch-gva-2,ch-dk-2,at-vie-1,de-fra-1}]
+                          [--rain RAIN]
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --n N                 Number of virtual machines
+  --name NAME           Virtual machine name prefix
+  --env ENV             Path to environment file
+  --keypair {vojtech,spirali}
```

Comment:
This line can be quite confusing for a general user, since these names (vojtech & spirali) go from our particular Exoscale account (am I right?).
Btw: Do we really need copy&pasted text here that can be simply obtained by running command with -h?

Reply:
I find it quite useful to have it available also here. Nevertheless, I replaced the names with more general names (key1, key2)...